Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SwiftTryCatch",
+    platforms: [
+        .iOS(.v13),
+        .tvOS(.v13),
+        .watchOS(.v6),
+        .macOS(.v10_15),
+    ],
+    products: [
+        .library(
+            name: "SwiftTryCatch",
+            targets: ["SwiftTryCatch"]),
+    ],
+    targets: [
+        .target(
+            name: "SwiftTryCatch",
+            dependencies: [],
+            publicHeadersPath: "./"),
+        .testTarget(
+            name: "SwiftTryCatchTests",
+            dependencies: ["SwiftTryCatch"])
+    ]
+)
+let version = Version(1, 0, 0)
```

Comment:
Per https://github.com/williamFalcon/SwiftTryCatch/issues/17#issuecomment-396191831, let me know if you'd like me to bump the version:
```suggestion
let version = Version(1, 0, 1)
```

Reply:
Looks like I'll need to bump the version to publish this package with Swift Package Manager, unless we want to re-tag the `1.0.0` tag on this repo.